### DISCUSSION TITLE
Server

### DISCUSSION THREAD

**Discussion by realchlorum** [PhilippvK/playforia-minigolf#80] [General]
Hello,

i want this game on my webserver with a domain.
So i can play this game easier with my friends.
Is there a way to contact you in any way? Email, Telegream etc.?

**Comment by PhilippvK** [comment]
Choose any of the contact options you can find on philippvk.github.io. I would prefer LinkedIn, Xing or E-Mail.

**Reply by PhilippvK** [reply]
But one information in advance: Java Applet running in the Webbrowser are officially dead. You could possible run them with a lot of manual steps in a 2 years old version of Firefox but this is not recommended and also far away from “easy to use”. Hosting a server is in fact possible and once it runs 24/7, you do not have to dead with stuff like Port Forwarding or LogMeIn Hamachi. Client side binaries are required in all possible cases. An outdated release has a Windows installer with comes with an integrated JRE so it does not require a Java installation but we can only offer the client.jar and client.exe for future releases due to an update build process.

**Comment by jirkovoriginal** [comment]
Hi.
 I started a server for anyone interested in playing multiplayer but can't have their own server. Just follow the simple instructions on the [page here](https://jirkov.tk/minigolf/) then just enjoy playing with your friends.

**Reply by realchlorum** [reply]
Awesome thanks works fine

**Reply by Arin-broke-it** [reply]
Hey @jirkovoriginal my friend didn't read the instructions on your site and selected the all maps option, now it's not working.  Does it now require a restart on your end?  How difficult is it to set up a server?

**Reply by jirkovoriginal** [reply]
Hi @Arin-broke-it , the server restarts itself every day at 3:30 CET, so there's nothing to worry about and if an error occurs it's unfortunately sad, because if no one tells me about it, I won't know about it, but the next day after the automatic restart, everything is working as it should .

Otherwise, starting the server is very easy. Just download the server.jar and the tracks folder (unpack it), put in in same directory and just run it like a regular program written in Java.

Finally, I made the font bigger and made the warning red to make it easier to see and hard to miss.

**Reply by Arin-broke-it** [reply]
Awesome, thanks!  The server setup is a little beyond my expertise but maybe one of my more tech-inclined friends can set it up haha

Do you know if there's any way to get the right-click aim adjustor to work on my macbook?  Seems to work fine for my friends on PC but not working for me unfortunately

edit:  I just figured it out, gotta hit command+click and it'll work.  Figure'd I'd post it here in case any future mac users need to know!

**Reply by realchlorum** [reply]
thanks for running the server. We are playing everyday :)
